src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNet/DbCommandIntegration.cs

```diff
@@ -249,21 +338,69 @@ public static class DbCommandIntegration
             int opCode,
             int mdToken,
             long moduleVersionPtr)
+        {
+            return ExecuteNonQueryInternal(
+                command,
+                opCode,
+                mdToken,
+                moduleVersionPtr,
+                methodName: AdoNetConstants.MethodNames.ExecuteNonQuery,
+                returnTypeName: ClrNames.Int32);
+        }
+
+        /// <summary>
+        /// Instrumentation wrapper for explicit implementation of <see cref="IDbCommand.ExecuteNonQuery()"/> in <see cref="DbCommand"/>.
+        /// </summary>
+        /// <param name="command">The object referenced by this in the instrumented method.</param>
+        /// <param name="opCode">The OpCode used in the original method call.</param>
+        /// <param name="mdToken">The mdToken of the original method call.</param>
+        /// <param name="moduleVersionPtr">A pointer to the module version GUID.</param>
+        /// <returns>The value returned by the instrumented method.</returns>
+        [InterceptMethod(
+            TargetAssemblies = new[] { AdoNetConstants.AssemblyNames.SystemData, AdoNetConstants.AssemblyNames.SystemDataCommon, AdoNetConstants.AssemblyNames.NetStandard },
+            TargetMethod = AdoNetConstants.MethodNames.ExecuteNonQueryExplicit,
```

Comment:
This doesn't show up in `System.Data, Version=4.0.0.0` in the GAC. Does this show up in OOB NuGet packages?

Reply:
Nope, I think you're right. I only see the `ExecuteReader()` overloads with explicit interface implementations.

Reply:
I removed all the attempts at instrumenting the explicit implementations since that's not going to work that way I thought. Thanks! 4b3f9747f70aad993f6f7e36dbb1426c48e04879